syft/frameworks/torch/tensors/interpreters/abstract.py

```diff
@@ -12,18 +15,28 @@ class AbstractTensor(ABC):
 
     is_wrapper = False
 
-    def __init__(self, tags=None, description=None):
+    def __init__(self, id=None, owner=None, tags=[], description=None, child=None, parent=None):
```

Comment:
Is parent a thing now? if so what does it do and how does this affect garbage collection, seems like it would create a reference loop

Reply:
It already a thing, it exists but we don't really make use of it
We could discuss removing it

Reply:
Sounds good. I will add an issue for it